src/cpuprofiler.js

```diff
@@ -297,7 +297,11 @@ var emscriptenCpuProfiler = {
     // you can manually create this beforehand.
     var cpuprofiler = document.getElementById('cpuprofiler');
     if (!cpuprofiler) {
-      var css = '.colorbox { border: solid 1px black; margin-left: 10px; margin-right: 3px; display: inline-block; width: 20px; height: 10px; }  .hastooltip:hover .tooltip { display: block; } .tooltip { display: none; background: #FFFFFF; margin-left: 28px; padding: 5px; position: absolute; z-index: 1000; width:200px; } .hastooltip { margin:0px; }';
+      cpuprofiler = document.createElement("div");
+      cpuprofiler.id = 'cpuprofiler';
+      document.body.appendChild(cpuprofiler);
+    }
+    var css = '.colorbox { border: solid 1px black; margin-left: 10px; margin-right: 3px; display: inline-block; width: 20px; height: 10px; }  .hastooltip:hover .tooltip { display: block; } .tooltip { display: none; background: #FFFFFF; margin-left: 28px; padding: 5px; position: absolute; z-index: 1000; width:200px; } .hastooltip { margin:0px; }';
```

Comment:
It looks like the above `}` is maybe misplaced? This causes the indentation then to not match further down.